site/app/templates/polls/AllPollsPageInstructor.twig

```diff
@@ -29,31 +34,12 @@
                 {% for poll in todays_polls %}
                     <tr>
                         <td> {{ poll.getName() }} </td>
-                        {% if poll.isOpen() %}
-                            <td>
-                                <form method="post" action="{{ base_url }}/setEnded">
-                                    <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
-                                    <input type="hidden" name="poll_id" value="{{ poll.getID() }}"/>
-                                    <button type="submit" class="btn btn-primary">End Poll</button>
-                                </form>
-                            </td>
-                        {% elseif poll.isClosed() %}
-                            <td>
-                                <form method="post" action="{{ base_url }}/setOpen">
-                                    <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
-                                    <input type="hidden" name="poll_id" value="{{ poll.getID() }}"/>
-                                    <button type="submit" class="btn btn-primary">Open Poll</button>
-                                </form>
-                            </td>
-                        {% elseif poll.isEnded() %}
-                            <td>
-                                <form method="post" action="{{ base_url }}/setOpen">
-                                    <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
-                                    <input type="hidden" name="poll_id" value="{{ poll.getID() }}"/>
-                                    <button type="submit" class="btn btn-primary">Reopen Poll</button>
-                                </form>
-                            </td>
-                        {% endif %}
+                        <td>
+                            <input class="key_to_click" tabindex="0" type="checkbox" id="poll_{{ poll.getID() }}_visible" aria-label="label" onclick='updatePollVisible("{{ poll.getID() }}")' {{ (poll.isOpen() or poll.isEnded) ? 'checked' : '' }}>
```

Comment:
You can remove the `key_to_click` class here, HTML5 checkboxes will allow users to select them by tabbing and toggling their checkboxes with the enter key by default